<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrachenflugDialog</class>
 <widget class="QDialog" name="DrachenflugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>270</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>217</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_ziel">
        <property name="text">
         <string>Ziel :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Abf">
        <property name="text">
         <string>Abfahrt :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Ank">
        <property name="text">
         <string>Ankunft :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Preis">
        <property name="text">
         <string>Preis :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ent">
        <property name="text">
         <string>Entfernung :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_2"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
